content/language-agnostic/encyclopedia/what-is-a-comment.md

```diff
@@ -6,27 +6,28 @@ Subjects:
 Tags: 
   - "comment"
   - "documentation"
+  - "best practice"
 Catalog Content: 
   - "https://www.codecademy.com/learn/learn-python-3"
   - "https://www.codecademy.com/learn/paths/code-foundations"
 ---
 
-A _comment_ is a note or explanation in the source code of a computer program. They are added with the purpose of making the code easier for future selves or other fellow developers to understand, and are generally ignored by compilers and interpreters. 
+A _comment_ is a note or explanation in the source code of a computer program. They are added with the purpose of making the code easier for ourselves or other developers to understand in the future, and they are generally ignored by compilers and interpreters. 
 
-Comments are generally formatted as either:
+Comments are typically formatted as either:
 
-- _Single-line comments_ start with a comment delimiter and continue until the end of the line.
-- _Multi-line comments_ start with a start delimiter and end with an end delimiter and can span multiple lines.
+- _Single-line comments_, which start with a comment delimiter and continue until the end of the line.
+- _Multi-line comments_, which start with a start delimiter and end with an end delimiter and can span multiple lines.
 
-Some programming languages support only one type of comment. For example, Python comments are single-line comments: they start with `#` and continue to the end of the line. Other languages employ both single-line and multi-line comments. For example, C and C++ have single-line comments created by `//` and multi-line comments between `/*` and `*/` that can span multiple lines.
+Some programming languages support only one type of comment. For example, Python comments are single-line comments: They start with `#` and continue to the end of the line. Other languages employ both single-line and multi-line comments. For example, C and C++ have single-line comments that start with `//` and multi-line comments between `/*` and `*/` that can span multiple lines.
 
 ## Uses
 
 How best to make use of comments is subject to dispute. Here are some common ways comments are used:
 
 ### Code Description
 
-Comments can be used to summarize code or to explain the programmer's intent. According to this school of thought, restating the code in plain English is considered superfluous; the need to re-explain code may be a sign that it is too complex and should be rewritten, or that the naming is bad.
+Comments can be used to summarize code or to explain the programmer's intent. According to this school of thought, restating the code in plain English is superfluous; the need to re-explain code may be a sign that it is too complex and should be rewritten, or that the naming is bad.
 
 > "Good comments don't repeat the code or explain it. They clarify its intent. Comments should explain, at a higher level of abstraction than the code, what you're trying to do." (Steve McConnell)
 
```

Comment:
I can't comment on line 35 (per the original file) because I didn't edit it, but I wanted to flag this thought that I wrote down when I read it: "How so? Could use some more details." Essentially, it isn't clear exactly _how_ comments can be used for debugging.

Reply:
edited line 35 (added "through process of elimination")